star/make/makefile_base

```diff
@@ -254,6 +254,7 @@ clean:
 	-@rm -f *.f90 *.o *.mod *.smod *genmod.f90 *.so *.a .depend .order
 
 install:
+	@$(CP_IF_NEWER) pgstar*.mod $(MESA_DIR)/include
```

Comment:
I'm going to try and come up with a better solution for giving binary access to the pgplot code instead of use just copying everything out of star.

Reply:
Okay i couldn't come up with a better solution without rewriting alot of the pgplot code